boards/common/remote/Makefile.include

```diff
@@ -11,6 +11,9 @@ ifneq (,$(PORT_BSL))
 endif
 
 ifeq ($(PROGRAMMER),cc2538-bsl)
+  PROG_BAUD ?= 115200
+  include $(RIOTMAKE)/tools/cc2538-bsl.inc.mk
+
   RESET ?= $(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py -p "$(PROG_DEV)"
   FLASHER = $(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py
   FLASHDEPS += $(FLASHER)
```

Comment:
You can also delete those

Reply:
Ah, yes thanks, I just forgot to remove the lines...